### ISSUE TITLE
rootpy.ROOT TFile wrapper problem

### ISSUE THREAD

**Issue by Awen0121** [rootpy/rootpy#702]
```python
with ROOT.TFile("./a.root","recreate") as of:
    pass
```
```python
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "/usr/lib/python2.7/site-packages/rootpy/io/file.py", line 225, in __exit__
    self.Close()
  File "/usr/lib/python2.7/site-packages/rootpy/io/file.py", line 254, in Close
    return self.cd_previous()
  File "/user/lib/python2.7/site-packages/rootpy/io/file.py", line 232, in cd_previous
    if isinstance(self._prev_dir, ROOT.TROOT):
  File "/usr/lib/python2.7/site-packages/rootpy/io/file.py", line 188, in __getattr__
    "{0} has no attribute '{1}'".format(self, attr))
AttributeError: File('./a.root') has no attribute '_prev_dir'
```
I simply add:
```python
#rootpy/io/file.py: line 248
     def Close(self, *args):
        """
        Like ROOT's Close but reverts to the gDirectory before this file was
        opened.
        """
        super(_DirectoryBase, self).Close(*args)

        - return self.cd_previous()
        + try:
        +     return self.cd_previous()
        + except AttributeError:
        +     return False
```
to make it work now. Hope it can get fixed before long.

**Comment by ndawe** [comment]
You must be using rootpy's ROOT wrapper like this:

```python
from rootpy import ROOT

with ROOT.TFile("./a.root","recreate") as of:
    pass
```

I'll fix that bug, but instead it would be more ideal to do the following:

```python
from rootpy.io import root_open

with root_open("./a.root", "recreate") as of:
    pass
```

**Comment by ndawe** [comment]
Fixed with #703
